data-raw/bucharest.R

```diff
@@ -8,7 +8,13 @@ bbox_buffer <- 2000
 bbox <- get_osm_bb(city_name)
 bbox_expanded <- buffer_bbox(bbox, bbox_buffer)
 bucharest_osm <- get_osmdata(bbox_expanded, city_name, river_name,
-                             crs = epsg_code)
+                             crs = epsg_code, force_download = TRUE)
+
+# Add delineation to package data
+network <- rbind(bucharest_osm$streets, bucharest_osm$railways) |> as_network()
+bucharest_delineation <-
+  # TODO this should be done with `delineate_corridor()` instead
+  list(corridor = delineate(city_name, river_name, crs = epsg_code))$corridor
```

Comment:
If we want to add the delineated corridor, why not to add all the delineations? I think this is also planned looking at the docstring of `bucharest_delineation` in `R/data.R`?
```suggestion
bucharest_delineation <- delineate(city_name, river_name, crs = epsg_code, 
                                   corridor = TRUE, segments = TRUE, 
                                   riverspace = TRUE)
```

If you think instead that we should only keep the corridor, I think there is a bracket issue:

```suggestion
network <- rbind(bucharest_osm$streets, bucharest_osm$railways) |> as_network()
bucharest_delineation <-
  # TODO this should be done with `delineate_corridor()` instead
  list(corridor = delineate(city_name, river_name, crs = epsg_code)$corridor)
```